connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/MirrorClient.java

```diff
@@ -60,7 +60,7 @@
     private ReplicationPolicy replicationPolicy;
     private Map<String, Object> consumerConfig;
 
-    public MirrorClient(Map<String, Object> props) {
+    public MirrorClient(Map<String, ?> props) {
```

Comment:
This is an incidental API change, but I don't _think_ it is a breaking change.

Reply:
Is this actually needed?

Reply:
i thought it was but i must have fixed something cuz it's compiling fine without it now :)